Gopkg.lock

```diff
@@ -580,6 +580,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "6c5bb30b3fc2f361b83a245f51cd824ba49f4e46a125ca38702c8b561cd5dd83"
+  inputs-digest = "62a12e75f5497a585e2cf71c56e6129dbdec6d69886bb10e4346eb67008de949"
```

Comment:
why is this change required?

Reply:
this is autogenerated

Reply:
You can revert this change, I think it is changed because of external lib used earlier

Reply:
sure